libs/core/src/calendar/internals/useUtils.ts

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { PickersAdapterContext } from '../LocalizationProvider';
+import { PickersAdapterContextValue } from '../LocalizationProvider/LocalizationProvider';
+
+export const useLocalizationContext = <TDate>() => {
+  const localization = React.useContext(PickersAdapterContext);
+
+  if (localization === null) {
+    throw new Error(
+      [
+        'FrontZen: Can not find the date and time pickers localization context.',
+        'It looks like you forgot to wrap your component in LocalizationProvider',
+      ].join('\n'),
+    );
+  }
+
+  return localization as PickersAdapterContextValue<TDate>;
+};
+
+export const useUtils = <T>() => useLocalizationContext<T>().utils;
+export const useLocaleText = <T>() => useLocalizationContext<T>().localeText;
+
+export const useNow = <TDate>(): TDate => {
+  const utils = useUtils<TDate>();
+  const now = React.useMemo(() => utils.date()!, [utils]);
```

Comment:
`now` can be null. do you can use 
```suggestion
  const now = React.useMemo(() => utils.date() ?? new Date(), [utils]);
```
or something like that?

Reply:
We can't do that?
because the date object should be of type TDate.
new Date() is of JS Date type.